modules/@ergonode/media/src/components/Buttons/LinkRelationButton.vue

```diff
@@ -59,33 +77,27 @@ export default {
             const elements = href.split('/');
             const id = elements[elements.length - 1];
 
-            switch (this.namespace) {
-            case this.namespaces.PRODUCT:
+            if (!isEmpty(this.routeLinks) && this.routeLinks[this.namespace]) {
                 this.$router.push({
-                    name: PRODUCTS_ROUTE_NAME.PRODUCT_EDIT_GENERAL,
+                    name: this.routeLinks[this.namespace],
                     params: {
                         id,
                     },
                 });
-                break;
-            case this.namespaces.TEMPLATE:
-                this.$router.push({
-                    name: PRODUCT_TEMPLATE_ROUTE_NAME.PRODUCT_TEMPLATE_EDIT_GENERAL,
-                    params: {
-                        id,
-                    },
-                });
-                break;
-            default:
-                this.$router.push({
-                    name: ATTRIBUTES_ROUTE_NAME.ATTRIBUTE_EDIT_GENERAL,
-                    params: {
-                        id,
-                    },
+            } else {
+                this.$addAlert({
+                    type: ALERT_TYPE.ERROR,
+                    message: 'Unknown relation type',
                 });
-                break;
             }
         },
     },
 };
 </script>
+<style lang="scss" scoped>
+    .link-button--disabled {
```

Comment:
Button should have implemented disable state in it selfs. If it doesn't match the styling, it should be changed in core